packages/cascara/src/structures/hooks/useThemeCustomProperties.js

```diff
@@ -67,10 +67,13 @@ const addCustomProperties = (color) => {
 const removeCustomProperties = () =>
   document.getElementById(STYLE_IDENTIFIER).remove();
 
-const useThemeCustomProperties = ({ color }) => {
+const useThemeCustomProperties = ({ color } = {}) => {
```

Comment:
```suggestion
const useThemeCustomProperties = ({ color }) => {
```
I _think_ we don't need to do this because if we do the check for color on line 73, and color is undefined, it should evaluate to false. I am pretty sure the change we need is just to check for `color` before we try to use it in the `addCustomProperties` function, right?

Reply:
@rociobaena I think this is the only thing we need to fix. Otherwise LGTM. Please @ me if this change gets made, or if this still fails without defining an empty object as default.

Reply:
Yes, that's right @brianespinosa. Let me play with that just a little with that, and then I will update the PR